src/editors/data/redux/thunkActions/video.js

```diff
@@ -32,9 +33,9 @@ export const loadVideoData = () => (dispatch, getState) => {
     allowTranscriptDownloads: rawVideoData.download_track,
     showTranscriptByDefault: rawVideoData.show_captions,
     duration: { // TODO duration is not always sent so they should be calculated.
-      startTime: rawVideoData.start_time,
-      stopTime: rawVideoData.end_time,
-      total: null, // TODO can we get total duration? if not, probably dropping from widget
+      startTime: valueFromDuration(rawVideoData.start_time || '00:00:00'),
+      stopTime: valueFromDuration(rawVideoData.end_time || '00:00:00'),
+      total: 0, // TODO can we get total duration? if not, probably dropping from widget
```

Comment:
remove this todo

Reply:
this might be a future enhancement (getting the real video total from the backend, which we don't have yet), so I'm tempted to leave it in.